esp/xslt/esdl2monitor.xslt

```diff
@@ -227,12 +232,12 @@ END;
 // we start to throttle, and maxRetries controls how many times inserts that fail because Cassandra is too busy
 // will be retried.
 
-monitorStoreRec getStoredMonitor(string id) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace('eclresultmonitoring'))
-  SELECT monitorId, result from monitors WHERE monitorId=? LIMIT 1;
+monitorStoreRec getStoredMonitor(string id) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace(csndKeySpaceFrom))
+  SELECT monitorId, result from monitor WHERE monitorId=? LIMIT 1;
 ENDEMBED;
 
-updateMonitor(dataset(monitorStoreRec) values) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace('eclresultmonitoring'), maxFutures(100), maxRetries(10))
-  INSERT INTO monitors (monitorId, result) values (?,?);
+updateMonitor(dataset(monitorStoreRec) values) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace(csndKeySpaceTo), maxFutures(100), maxRetries(10))
+  INSERT INTO monitor (monitorId, result) values (?,?);
```

Comment:
this is just how the embedded cassandra works.  If the parameter is a dataset the items of the rows are used in order.